local_storage/dockerize_kafka.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# TODO: rewrite with docker-compose?
```

Comment:
Yes, please, we already has docker-compose with kafka https://github.com/RedHatInsights/ccx-data-pipeline/blob/master/docker-compose.yml